From 1277b5a8a578c262f259034db7c7d4adc81c82de Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 26 Dec 2024 10:13:42 +0800
Subject: [PATCH 138/258] AOSCOS: platform: ideapad-laptop: fix if branch
 syntax

With "AOSCOS: platform: x86: ideapad-laptop: add fixes for ThinkBook 14+
(2024 G6+)", an if branch guarding a no-data condition was introduced:

  if(!data)
  	acpi_handle_info(handle, "no data");
  	return;

However, it was missing braces as it's a multi-line branch.

Add a pair of braces to fix this syntax error.

Fixes: "AOSCOS: platform: x86: ideapad-laptop: add fixes for ThinkBook 14+ (2024 G6+)"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/platform/x86/ideapad-laptop.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/platform/x86/ideapad-laptop.c b/drivers/platform/x86/ideapad-laptop.c
index 8a59cd421bf8..31836939ecda 100644
--- a/drivers/platform/x86/ideapad-laptop.c
+++ b/drivers/platform/x86/ideapad-laptop.c
@@ -1863,9 +1863,10 @@ static void ideapad_acpi_notify(acpi_handle handle, u32 event, void *data)
 		acpi_handle_info(handle, "event: %lu\n",
 						(unsigned long)event);
 
-		if(!data)
+		if(!data) {
 			acpi_handle_info(handle, "no data");
 			return;
+		}
 
 		if (priv->suspended)
 			return;
-- 
2.48.1

